### PR TITLE
Update main.tf

### PR DESCRIPTION
**Pull request by deepakagrawal2**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -40,6 +40,9 @@ resource "aws_db_instance" "main_rds_instance" {
 
     backup_retention_period = "${var.backup_retention_period}"
     backup_window = "${var.backup_window}"
+    tags {
+        "Name" = "Test"
```
Comment (gstlt) [inline main.tf:44]: Quotes around "Name" are not needed.
Also: How this is useful? Maybe use ${var.database_name} or something?

Comment (ghost) [inline main.tf:44]: @gstlt Same comment here, this tag does not provide any value.
@deepakagrawal2 Are you going to parameterize `Name` tag?

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by af6140** [comment]
Mos likely just an accident with git operation.

**Comment by antonbabenko** [comment]
This PR does not bring anything new, so I closed it.

**Pull request closed without merging**